hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/missingContainers/missingContainers.tsx

```diff
@@ -244,32 +229,30 @@ export class MissingContainers extends React.Component<Record<string, object>, I
     });
 
     axios.all([
-      axios.get('/api/v1/containers/unhealthy/MISSING'),
-      axios.get('/api/v1/containers/unhealthy/UNDER_REPLICATED'),
-      axios.get('/api/v1/containers/unhealthy/OVER_REPLICATED'),
-      axios.get('/api/v1/containers/unhealthy/MIS_REPLICATED')
-    ]).then(axios.spread((missingContainersResponse, underReplicatedResponse, overReplicatedResponse, misReplicatedResponse, allReplicatedResponse) => {
+      axios.get('/api/v1/containers/unhealthy')
+    ]).then(axios.spread((allReplicatedResponse) => {
+
+      const allContainersResponseData: IUnhealthyContainersResponse = allReplicatedResponse.data;
+      const allContainers: IContainerResponse[] = allContainersResponseData.containers;
```

Comment:
just curious~ what does 'I' in IContainerResponse mean?

Reply:
Interface